### PR TITLE
Dark mode for tab

### LINKED ISSUE #6577: Gradient needs to work in light mode
image

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Fixes #6577

Now:

image

image

### CODE REVIEW

**Review by jasonLaster — APPROVED** [review]

File: src/ui/components/SecondaryToolbox/SecondaryToolbox.css [hunk 1]
```diff
@@ -112,5 +112,5 @@ overflows its container (https://drafts.csswg.org/css-flexbox-1/#min-size-auto)
   left: -2rem;
   width: 2rem;
   height: 100%;
-  background-image: linear-gradient(to left, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+  background-image: linear-gradient(to left, var(--tab-bgcolor), rgba(0, 0, 0, 0));
```
Comment (bvaughn) [inline src/ui/components/SecondaryToolbox/SecondaryToolbox.css:115]: I think fading from white to `rgba(0, 0, 0, 0)` will cause not just the opacity but the color to fade.

Comment (jonbell-lot23) [reply src/ui/components/SecondaryToolbox/SecondaryToolbox.css:115]: Hm -- it feels pretty good to me, am I missing something?

**Comment by jonbell-lot23** [comment]
I'm going to go with Brian's PR instead of this one.

File: src/ui/components/SecondaryToolbox/SecondaryToolbox.css (hunk 1 above)
Comment (bvaughn) [reply src/ui/components/SecondaryToolbox/SecondaryToolbox.css:115]: Interesting! I wonder if different browsers treat this differently. Or maybe I'm mistaken on how the fade works when there is a color _and_ opacity shift! My apologies.

**Pull request closed without merging**